src/Controller/QuestionController.php

```diff
@@ -37,4 +44,40 @@ public function index(): string
             'errors' => $errors]
         );
     }
+
+    /**
+     * Displays a message that tells if the user answered the question correctly.
+     * Redirects to index if $_POST['choice'] does not exist or is empty.
+     * @return string
+     */
+    public function result(): string
+    {
+        $message = "";
+        $choiceManager = new ChoiceManager();
+
+        // Check if choice exists and is an int !==0
+        if (isset($_POST['choice'])  && intval($_POST['choice']) !== 0) {
+            $id = $_POST['choice'];
+            $allChoices = $choiceManager->selectAllIds();
+
+            // Check if choice id exists in database
+            if (in_array($id, $allChoices)) {
```

Comment:
It's probably a better approach to select the choice with the given id a maybe get no result, than getting all the database then check if our value is in it.